### PR TITLE
Added norwegian nynorsk🇳🇴

### PR DESCRIPTION
**Pull request by petrepa**
Also changed Norwegian bokmål from `no` to `nb` to fit the ISO 639-1 Code. Norwegian nynorsk
 is under `nn`.

 Should we maybe change the language section in the README to list the
 langauges in an unordered list or something to make it clearer what
 languages the codes represent?

To maybe something like this?
- `da`: Danish
- `de`: German
- `en`: English
- `es`: Spanish
- `et`: Estonian
- `fi`: FInnish 
- `fr`:  French
- `hu`: Hungarian
- `it`: Italian
- `nb`: Norwegian (bokmål)
- `nl`: Dutch
- `nn`: Norwegian (Nynorsk)
- `pl`: Polish
- `pt-BR`: Portuguese (Brazil)
- `ru`: Russian
- `sk`: Slovak
- `sl`: Slovenian
- `sv`: Swedish

(Cannot guarantee that all of the codes are correct tho)

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -7,7 +7,9 @@ import fi from './assets/localization/languages/fi.json'
 import fr from './assets/localization/languages/fr.json'
 import hu from './assets/localization/languages/hu.json'
 import it from './assets/localization/languages/it.json'
+import nb from './assets/localization/languages/nb.json'
```
Comment (bjornomy) [inline src/constants.ts:11]: Not sure if there are collisions, but we should probably adhere to the normal locale standard `no_nn / no_nb`, like done for `pt-br`

Comment (petrepa) [reply src/constants.ts:11]: Doesn't matter to me😊  Preferably just pick a standard and stick to it

Comment (AitorDB) [reply src/constants.ts:11]: Although I agree that ideally we should use the same standard for everything, that seems to not be the case for Home Assistant (or they are using a standard that I don't know). I just checked changing the languages on Home Assistant and the values returned for both cases:

- Norsk Bokmål = `nb`
- Norsk Nynorsk = `nn`

So if we want language to work directly with Home Assistant we are going to have to follow what they use, hopefully that's not an issue when using some of the browser methods to adapt times and more

**Review by AitorDB — APPROVED** [review]

File: src/constants.ts (hunk 1 above)
Comment (AitorDB) [reply src/constants.ts:11]: It seems to be a bit more complex and it's using multiple standards https://stackoverflow.com/a/21853099

Comment (AitorDB) [reply src/constants.ts:11]: @bjornomy GH is not letting me to request for a re-review from you (not sure why), but feel free to add any feedback to the conversation or approve and I'll merge it

Comment (bjornomy) [reply src/constants.ts:11]: As a norwegian my self, I've dealt with this unique problem before (very few nations has more than 1 offical written language), and that's why I mentioned it.

After my rewrite in #64, any approach should work, as `Intl.DateTimeFormatter` does some magic here.
I tested using `nn` and `nb` as language, and it does work on my branch. As such we could probably use those.

**Comment by AitorDB** [comment]
@petrepa sorry it took this long, I wanted to merge #64 before. There's a few conflicts, do you mind to take a look to it?

**Comment by petrepa** [comment]
@AitorDB No worries! Think we should be good to go now!

**Comment by AitorDB** [comment]
Awesome! Thanks!

**Pull request merged**